docs/upgrade_1-2_to_1-3.md

```diff
@@ -1,21 +1,23 @@
 # Upgrading from OADP 1.2
 
-> **NOTE:** Always upgrade to next minor version, do NOT skip versions. To update to higher version, do one upgrade at a time. Example: to upgrade from 1.1 to 1.3, upgrade first to 1.2, then to 1.3.
+> **NOTE:** Always upgrade to next minor version, do NOT skip versions. To update to higher version, please upgrade one channel at a time. Example: to upgrade from 1.1 to 1.3, upgrade first to 1.2, then to 1.3.
 ## Changes from OADP 1.2 to 1.3
 
-- Velero was updated from version 1.11 to 1.12 (Changes reference: https://velero.io/docs/v1.12/upgrade-to-1.12/#upgrade-from-v110-or-higher)
+- The Velero server has been updated from version 1.11 to 1.12 (Changes reference: https://velero.io/docs/v1.12/upgrade-to-1.12/#upgrade-from-v110-or-higher)
 
-    From this update, OADP now uses Velero Built-in DataMover instead of VSM/Volsync DataMover. This change the following:
+    From this update, OADP 1.3 now uses the Velero Built-in Data Mover instead of the VSM/Volsync DataMover. This changes the following:
```

Comment:
there is both `DataMover` and `Data Mover` in the file. Would use just one wording in whole file

Reply:
resolved